src/python/zquantum/qaoa/parameter_initialization.py

```diff
@@ -0,0 +1,161 @@
+from typing import List, Tuple
+
+import numpy as np
+from zquantum.core.interfaces.cost_function import ParameterPreprocessor
+
+
+def _get_param(params_vector: List[float], layer_number):
+    """0 <= layer_number <= p + 1"""
+    if layer_number == 0 or layer_number == len(params_vector) + 1:
+        return 0
+    else:
+        return params_vector[layer_number - 1]
+
+
+def _perform_1_interpolation(
```

Comment:
[suggestion] I don't like use of `1` here. Not a big deal, but I would just use `one`.

Reply:
Fixed in [a716e5b](https://github.com/zapatacomputing/z-quantum-qaoa/pull/84/commits/a716e5bcd72f10a56661bd73d310b9b00ecb1dd0)

Reply:
Actually, now that I look at it again, `single` would look even better, if you don't mind changing it again 😅

Reply:
Np!